Exemple page with class API not work

Hi, 

On this page : https://typescript.nuxtjs.org/examples/class-api/minimal/
The example on Class API not work with this error message : Cannot find module 'vue-class-component' 


**To Reproduce**
Go to this page : https://typescript.nuxtjs.org/examples/class-api/minimal/

**Expected behavior**
A good compilation example with for result : "This is a message"

--- a/examples/class-api/minimal/package.json
+++ b/examples/class-api/minimal/package.json
@@ -4,6 +4,7 @@
   "private": true,
   "dependencies": {
     "nuxt": "latest",
+    "vue-class-component": "latest",
     "vue-property-decorator": "latest"
   },
   "scripts": {

